GVFS/GVFS.Common/HealthCalculator/EnlistmentHealthCalculator.cs

```diff
@@ -10,18 +10,18 @@ namespace GVFS.Common
     /// internally stores the calculated information. Compute or recompute via CalculateStatistics
     /// with an optional parameter to only look for paths which are under the specified directory
     /// </summary>
-    public class GVFSEnlistmentHealthCalculator
+    public partial class EnlistmentHealthCalculator

```

Comment:
Why is this class now `partial`?

Reply:
Oh whoops that was a line I missed when doing the refactoring. I tried to use the built in tools to extract inner classes, but it just created a new file and left them as members of `EnlistmentHealthCalculator` but in new files, hence why it was automatically modified to be a partial class. I missed that line when I went back through to change what had been automatically refactored.

Reply:
I'll very quickly push that fix up.